libs/postgres_ffi/src/wal_craft.rs

```diff
@@ -152,17 +148,48 @@ impl Conf {
     }
 }
 
+fn create_unix_socket_directory() -> anyhow::Result<PathBuf> {
+    let current_millis = SystemTime::now()
+        .duration_since(UNIX_EPOCH)
+        .context("Faled to generate random tmp dir name")?
```

Comment:
Typo: `failed`

Reply:
And also did not comply the [recent discussion](https://neondb.slack.com/archives/C0277TKAJCA/p1673453329770429) results to keep the context out of parasite "Failed", "could not", etc. words.
Fixed.